install/jaxrs/bin/ts.jte

```diff
@@ -274,11 +259,11 @@ porting.ts.HttpsURLConnection.class.1=com.sun.ts.lib.implementation.sun.javaee.S
 
 
 #####################################################################
-## The following properties must be set prior to running the Jakarta RESTful Web Services
+## The following properties must be set prior to running the JAX-RS
```

Comment:
Use Jakarta RESTful Web Services instead of JAX-RS

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.